app/views/endemics/biosecurity-exception.njk

```diff
@@ -18,10 +18,11 @@
         <h1 class="govuk-heading-l">You cannot continue with your claim</h1>
 
         <p class="govuk-body">
-        You need to <a href="https://apply-for-an-annual-health-and-welfare-review.defra.gov.uk/apply/claim-guidance-for-farmers">ask the vet to complete a biosecurity assessment.</a> Once you have done this, you can start your claim again.
+        The vet should have done a <a href="https://apply-for-an-annual-health-and-welfare-review.defra.gov.uk/apply/claim-guidance-for-farmers">biosecurity assessment.</a> Once you have done this, you can start your claim again.
         </p>
 
-        <p class="govuk-body">If you have selected the wrong option, <a class="govuk-link" href={{backLink}}>choose again if the vet did a biosecurity assessment.</a></p>
+        <p class="govuk-body">If the vet did a biosecurity assessment, you’ll need to go back and change your answer.</p>
+        <p class="govuk-body"><a class="govuk-link" href={{backLink}}>If the vet did a biosecurity assessment, you’ll need to go back and change your answer.</a></p>
```

Comment:
I don't think this matches what your ticket says

Reply:
![Screenshot 2024-04-19 at 09 23 02](https://github.com/DEFRA/ffc-ahwr-farmer-claim/assets/14424567/d383a7cf-c812-4d32-9181-7bfc5894c418)

Reply:
done